README.md

```diff
@@ -2,6 +2,7 @@
 
 > The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service. 
 
+![screenshot](./app_screenshot.png)
 
 ## Built With
```

Comment:
- [ ] [OPTIONAL] In the getting started part, you actually stated how a user can get a copy of the project but you also need to state that the user needs to run `npm i` to get all the project dependencies and run `npm start` to run the project.

Reply:
Thanks for making note of that, have added your suggestion. 👍🏽